app/config.py

```diff
@@ -1,3 +1,9 @@
 import os
 
 is_test_environment = os.environ.get("TEST_ENV", False)
+
+link_ttl = int(os.environ.get("LINK_TTL", "600"))
+region_name = os.environ["REGION_NAME"]
+minio_endpoint = os.environ["MINIO_ENDPOINT"]
```

Comment:
Why do we need minio endpoint?

Reply:
You're right, we don't actually need the MinIO endpoint here. It was left over from a tutorial on boto3. I'll remove it